Source/Support/DispatchSource+Fisticuffs.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  DispatchSource+Fisticuffs.swift
+//  Fisticuffs
+//
+//  Created by Maksym Korytko on 2016-12-22.
+//  Copyright © 2016 theScore. All rights reserved.
+//
+
+import Foundation
+
+extension DispatchSource {
+    /**
+     Creates a single use timer source and resumes it.
+     Once the timer fires, it is automatically invalidated.
+     - parameters:
+        - queue: The dispatch queue that should be used to enque the handler.
+        - interval: Timer interval since now when the handler should be invoked.
```

Comment:
Just a couple tiny typos here: I think `enque` should be `enqueue` and `timer interval` should be `time interval` 🤓